src/arrayMethodReduce.js

```diff
@@ -4,9 +4,18 @@
  * Implement method Reduce
  */
 function applyCustomReduce() {
-  [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+  [].__proto__.reduce2 = function(callback, ...initialValue) {
```

Comment:
The usage of the ... operator can lead to incorrect results if more than one initial value is passed. It's better to use two separate arguments instead: function(callback, initialValue)

Reply:
@33Orange we discussed it with @alexander-ignatow  on Q&A session, that this is the most proper solution, because for tests we need to be able to reduce the array if initial value was set manually to undefined. Only 2 solutions we got are arguments or making initialValue parameter to be an array. As I remember coach said that arguments is distant solution and it is better not to use it, but I have to complete that test with undefined, so. If you think that proper decision is using arguments - I will refactor it.